hat/assets/js/apps/Iaso/domains/entities/components/BeneficiaryBaseInfo.tsx

```diff
@@ -54,30 +66,64 @@ export const BeneficiaryBaseInfo: FunctionComponent<Props> = ({
         ],
         [beneficiary?.attributes?.nfc_cards, beneficiary?.uuid, formatMessage],
     );
-    return (
-        <>
-            <Table size="small">
-                <TableBody>
-                    {!isFetchingForm && beneficiary && (
-                        <>
-                            {dynamicFields.map(field => (
-                                <PaperTableRow
-                                    label={field.label}
-                                    value={field.value}
-                                    key={field.key}
-                                />
-                            ))}
-                            {staticFields.map(field => (
-                                <PaperTableRow
-                                    label={field.label}
-                                    value={field.value}
-                                    key={field.key}
-                                />
-                            ))}
-                        </>
-                    )}
-                </TableBody>
-            </Table>
-        </>
-    );
+
+    if (isFetchingForm || !beneficiary) {
+        return (
+            <Skeleton
+                sx={{ marginBottom: '32px' }}
+                variant="rectangular"
+                width="100%"
+                height="30vh"
+            />
```

Comment:
❤️ 
Also you can straight use theme with sx:
marginBottom: theme => theme.spacing(4)